app/assets/javascripts/components/sidebar/profile_snippet_component.js.coffee

```diff
@@ -0,0 +1,8 @@
+window.ProfileSnippetComponent = React.createClass
+	render: ->
+		<div className="profile_container username_link" user_id={this.props.user.id}>
+			<img className="profilepic_small" src={this.props.user.img+"?sz=100"}></img>
+			<div className="ticket_name"> 
+				{ this.props.user.first } 
```

Comment:
I'm assuming this is the first name and not the first user in a list of users, but simply because I know the context. it would be more clear is this was renamed in the json to be `firstName`